### PR TITLE
base_import_async: properly register the module with connector framework

### PR DESCRIPTION
**Pull request by sbidoul**
Plus change README.md to README.rst

### CODE REVIEW

**Comment by guewen** [comment]
Thanks!
:+1:

**Comment by lmignon** [comment]
:+1:

**Comment by sebastienbeau** [comment]
:+1: thanks

File: base_import_async/README.rst [hunk 1]
```diff
@@ -36,14 +41,19 @@ Other modules may benefit from this infrastructure in the following way
    (`header`, `encoding`, `separator`, `quoting`,
    `use_connector`, `chunk_size`).
 
-## Known limitations
+Known limitations
```
Comment (pedrobaeza) [inline base_import_async/README.rst]: Use better the usual section "Known issues / Roadmap"

File: base_import_async/README.rst [hunk 2]
```diff
@@ -18,7 +22,8 @@ being imported is stored as an attachment to the job, making
 it easy to download it, fix it and run a new import, possibly
 in synchronous mode since the chunks are small.
 
-## Scope
+Scope
```
Comment (pedrobaeza) [inline base_import_async/README.rst]: Use the standard section "Usage"

**Comment by sbidoul** [comment]
@pedrobaeza I took care of your comments.

**Comment by pedrobaeza** [comment]
Thanks :+1:

**Pull request merged**